### DISCUSSION TITLE
Task List

### DISCUSSION THREAD

**Discussion by kadaan** [doudar/SmartSpin2k#91] [Q&A] [answered]
@doudar Do you have a set of tasks that you can add and tag for others to take on?

**Comment by doudar** [comment, marked as answer]
I do not. 

I can assign issues to users that's along the lines of what you're looking for. 

A quick google also yielded this. 
https://docs.github.com/en/github/managing-your-work-on-github/about-task-lists

**Reply by kadaan** [reply]
Cool.  I'm really just looking to see if there there are tasks you want to get done that are not represented by issues.  I'd like to help out as I can.

**Reply by doudar** [reply]
Wow! Awesome offer!

I’ll assign you some stuff then but don’t stress out about it and if you don’t want it please let me know!

I’m going to assume you’re willing to tackle web stuff based on your last PR.

**Reply by kadaan** [reply]
I'm a backend developer by profession, but can help with the UI some.
